<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>850</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔스퀘어</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MySearch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="post_listview">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>600</width>
      <height>650</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="news_list_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>179</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your News List</string>
    </property>
   </widget>
   <widget class="QLabel" name="webpage_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Webpage</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="page_spinBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>760</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="page_index_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>730</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Page Index</string>
    </property>
   </widget>
   <widget class="QComboBox" name="webpage_comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="go_to_website_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Go To
Website</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
